tools/__tasks__/validate/a11y/reporter.js

```diff
@@ -15,5 +15,5 @@ module.exports = result => {
         output = lines.join('\n');
     }
 
-    return output;
+    return output && output.length ? output : undefined;
```

Comment:
I think this could be simplified:

```js
if (typeCode <= logLevel) {
    return lines.join('\n');
}
```

Reply:
But then you'd end up with an array full of empty strings. This is why the `.length` check is introduced ...